examples/native/.storybook/main.js

```diff
@@ -1,4 +1,3 @@
 module.exports = {
-  //stories: ['./components/*/*.stories.?(ts|tsx|js|jsx)', './globtest/Buttonx/Buttonx.stories.tsx'],
-  stories: ['./components/Button/Button.stories.tsx'],
+  stories: ['./components/*/*.stories.?(ts|tsx|js|jsx)', './globtest/Buttonx/Buttonx.stories.tsx'],
```

Comment:
What's the purpose of `globtest/Buttonx/Buttonx.stories.tsx`?

Reply:
This is something that I did previously, I wanted to test the story detection was working for multiple different cases. I think the folder name "globtest" is a bit misleading actually. Both these strings are just specifying locations of story files. Could easily just remove this example, however it's to show you can specify a full path.

Reply:
fyi in this PR I just uncommented the line since I forgot to do so when I was testing the CSF example `['./components/Button/Button.stories.tsx'],` 😅.

Reply:
i chaged the name to be less strange :)